===================================================================
RCS file: /u/ijackson/public-CVS/adns/src/parse.c,v
retrieving revision 1.22
retrieving revision 1.23
diff -u -r1.22 -r1.23
--- adns/src/parse.c	2003/06/22 13:01:11	1.22
+++ adns/src/parse.c	2004/05/04 18:46:36	1.23
@@ -113,7 +113,7 @@
 }
 
 adns_status adns__parse_domain(adns_state ads, int serv, adns_query qu,
-			       vbuf *vb, adns_queryflags flags,
+			       vbuf *vb, parsedomain_flags flags,
 			       const byte *dgram, int dglen, int *cbyte_io,
 			       int max) {
   findlabel_state fls;
